applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,483 @@
+                        aria-controls={openMenu ? 'detailsViewer-customized-menu' : undefined}
+                        aria-haspopup="true"
+                        aria-expanded={openMenu ? 'true' : undefined}
+                        variant="text"
+                        disableElevation
+                        onClick={handleMenuClick}
+                        endIcon={<img src={DOWN_ICON} alt='' />}
+                    >
+                        <span className={classes.ellipsisText}>{data[tabIdx].files[selectedIndex].title}</span>
+                    </Button>
+                    <Menu
+                        id="detailsViewer-customized-menu"
+                        MenuListProps={{
+                            'aria-labelledby': 'demo-customized-button',
+                        }}
+                        className={classes.customMenu}
+                        anchorEl={anchorElMenu}
+                        open={openMenu}
+                        onClose={handleMenuClose}
+                    >
+                        <TextField
+                            placeholder="Search for a file"
+                            className={classes.searchField}
+                        />
+                        <Divider />
+                        {
+                            data[tabIdx].files.map((file: any, index) => (
+                                <div className={classes.menuItemBox} key={index}>
+                                    <MenuItem
+                                        disableGutters
+                                        selected={index === selectedIndex}
+                                        onClick={(event) => handleMenuItemClick(event, index)}
+                                    >
+                                        {file.title}
+                                    </MenuItem>
+                                    {
+                                        selectedIndex === index && <ListItemIcon>
+                                            <img src={CHECK} alt='' />
+                                        </ListItemIcon>
+                                    }
+                                </div>
+                            ))
+                        }
+                    </Menu>
+                    <Box>
+                        <Tooltip
+                            arrow
+                            title={<div style={{ display: 'flex', justifyContent: 'space-between', gap: '0.75rem' }}>
+                                <div className={classes.textBlock} style={{ color: sidebarTextColor }}>
+                                    <span>Uploaded on</span>
+                                    <span>Uploaded by</span>
+                                </div>
+                                <div className={classes.textBlock} style={{ alignItems: 'flex-end' }}>
+                                    <span>06 Nov 2023, 11.58am</span>
+                                    <span>Quinn Silverman</span>
```

Comment:
This is not correct. We should try to avoid having placeholder data hardcoded in the components. 
We should always try to think where will this data come from? Is the component making a request or is it going to be a prop?

Let's say it's a prop. We should add that prop to the component and mock that prop when calling it ( or using a default value ), like you did very well to the population color.